workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx

```diff
@@ -90,40 +90,30 @@ const MDFClaimPage = ({
 		return isDisplayable;
 	};
 
+	const availableMDFActivities = values.activities?.filter((activity) =>
+		isDisplayableMDFActivityClaim(activity)
+	).length;
+
+	const getCreateClaimDenialMessage = () => {
+		if (mdfRequest.mdfRequestStatus?.key !== 'approved') {
+			return 'Waiting for Manager approval.';
+		}
+		else if (claimsFiltered && claimsFiltered >= 2 && !values.id) {
+			return 'You already submitted 2 claims.';
+		}
+		else if (!availableMDFActivities || availableMDFActivities === 0) {
```

Comment:
check if we need to compare the same value twice

Reply:
Done!